utils/downloader.py

```diff
@@ -13,6 +20,10 @@
 DOWNLOAD_SESSION.mount('http://', forever_adapter)
 DOWNLOAD_SESSION.mount('https://', forever_adapter)
 
+gauth = GoogleAuth()
```

Comment:
Make all caps to make it clear these are constants, also might be good to add a comment on what they're used for